### ISSUE TITLE
Botw not reading

### ISSUE THREAD

**Issue by mbamber1986** [XorTroll/emuiibo#133]
Hi i dont know if its an issue not sure myself however i have found that that zelda botw does not read the emuibo files however my n2 elite device works fine.

i am currently using  ams 1.14.4
fw : 10.0.0.2
emuibo 1.6
i created the amibos with emuiibotool

emuiboo works witht he same virtuial amibo works fine on smash bros.

let me know if theres a fix or something im doing wrong or if i need to install something else

thanks in advanced.

**Comment by mbamber1986** [comment]
Fixed it I had an earlier firmware and was booting emuiibo  after game booted up

**Comment by mbamber1986** [comment]
So i fixed my first issue thanks to a fellow user on discord however I have found another issue this seems to be related to zelda

As stated before i have updated to the latest ams version  latest firmware and latest emuiibo  

The issue i have found Is that when I load the zelda botw I start emuiibo before hand and in handheld mode with joycons connected to the rails the amiibos  read fine it drop the items 

Take the joycon off the rail for docked mode or tablet on a stand mode the amiibos  done read it just gives some sort of rejection noise 

Now I havent done much extensive testing but the other game that supports amiibos is smash brothers  and on and off the rail this reads the emuiibo files

Which leads me to to think its either a game issue or a support for specific games or support for the games when off the rails.


Sorry for opening and closing the issue but I feel even though I sorted the first issue I felt this should be a adressed


Thanks again

**Comment by sunhch** [comment]
I'm having the same issue. I tried to reboot and delete everything emuiibo related and did a clean reinstall. I even used emutool to create some new emiibo files, but still had not luck. 
I wish I checked this issue page earlier, because it cost me hours to figure out what's going on...
Basically my switch has the same behavior. Namely, when it is in hand held mode, everything works fine, but when it is in dock mode, only around 1 out of 20 scans work.
Strangely, I remember when I tested emuiibo for the first time a few days ago (same emuiibo, OS, Atmosphere version), I used docked mode with pro controller and everything worked fine. Either I remember it wrong or there is some bug in emuiibo that caused dock mode to fail after the first BOTW session.
